### PR TITLE
Amdsmidocs reorg

### PR DESCRIPTION
**Pull request by Rmalavally**

### CODE REVIEW

File: docs/how-to/using-amdsmi.rst
Comment (randyh62) [inline docs/how-to/using-amdsmi.rst:1]: Not sure the rocDecode metadata is correct here.

Comment (randyh62) [inline docs/how-to/using-amdsmi.rst:1]: This provides a way to do any releasing of resources that AMD-SMI may have held.
-to-
This provides a way to release resources that AMD-SMI may have held.

Comment (randyh62) [inline docs/how-to/using-amdsmi.rst:1]: 1) A simple "Hello World" type program

Looks un-indented, while example 2 appears indented. They should either both be indented or not.

Comment (randyh62) [inline docs/how-to/using-amdsmi.rst:1]: Documentation
--------------

From this point down it appears that the topics are less about Using AMDSMI and more about the docs, and Building and Installing the repo. I think this content should appear in separate topics, like install.rst. 

In addition, as the reader is inside the documentation when reading this content, I don't know if we need to point to the PDF file documentation?

File: docs/sphinx/_toc.yml.in
Comment (randyh62) [inline docs/sphinx/_toc.yml.in:1]: It looks like your TOC does not have any API Reference files listed, although they do appear in the index.rst.

**Review by randyh62 — APPROVED** [review]
Left a few comments.

**Comment by Rmalavally** [comment]
> Left a few comments.

Thank you, Randy. This is still a work in progress. Appreciate your feedback though.

**Comment by Rmalavally** [comment]
> > Left a few comments.
> 
> Thank you, Randy. This is still a work in progress. Appreciate your feedback though.

Converted the PR to Draft.

**Review by lpaoletti — COMMENTED** [review]
Is there anything missing here?
image

**Comment by Rmalavally** [comment]
> Is there anything missing here? image

Likely but in the source content. This is Doxygen generated.

**Comment by marifamd** [comment]
Merged to develop

**Review by lpaoletti — APPROVED** [review]

**Comment by marifamd** [comment]
This is merged in

**Pull request closed without merging**